src/core/src/pass/sdpa_to_paged_attention.cpp

```diff
@@ -53,8 +53,21 @@ bool ov::pass::SDPAToPagedAttention::run_on_model(const std::shared_ptr<ov::Mode
 
     auto sliding_window = v0::Constant::create(element::i32, Shape{}, {0});  // sliding_window
 
+    auto has_parameter = [=](const std::shared_ptr<ov::Model>& model, const std::string& name) -> bool {
+        for (auto& t : model->inputs()) {
+            const auto& names = t.get_names();
+            if (names.find(name) != names.end()) {
+                return true;
+            }
+        }
+
+        return false;
```

Comment:
can we return the tensor we found or even already casted Parameter from this lambda not true/false?